tests/components/sfr_box/test_sensor.py

```diff
@@ -23,37 +21,30 @@ def override_platforms() -> Generator[None, None, None]:
         yield
 
 
-def _check_and_enable_disabled_entities(
-    entity_registry: er.EntityRegistry, expected_entities: MappingProxyType
-) -> None:
-    """Ensure that the expected_entities are correctly disabled."""
-    for expected_entity in expected_entities:
-        if expected_entity.get(ATTR_DEFAULT_DISABLED):
-            entity_id = expected_entity[ATTR_ENTITY_ID]
-            registry_entry = entity_registry.entities.get(entity_id)
-            assert registry_entry, f"Registry entry not found for {entity_id}"
-            assert registry_entry.disabled
-            assert registry_entry.disabled_by is er.RegistryEntryDisabler.INTEGRATION
-            entity_registry.async_update_entity(entity_id, **{"disabled_by": None})
-
-
 async def test_sensors(
     hass: HomeAssistant,
     config_entry: ConfigEntry,
     device_registry: dr.DeviceRegistry,
     entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
 ) -> None:
     """Test for SFR Box sensors."""
     await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
 
-    check_device_registry(device_registry, EXPECTED_ENTITIES["expected_device"])
+    device_entry = device_registry.async_get_device({(DOMAIN, "e4:5d:51:00:11:22")})
+    assert device_entry == snapshot(name="devices")
+
+    entity_entries = er.async_entries_for_config_entry(
+        entity_registry, config_entry.entry_id
+    )
+    assert entity_entries == snapshot(name="entities")
```

Comment:
Same as above, in this case for the entity entry object.

```suggestion
    assert entity_entries == snapshot
```